h/buildext.py

```diff
@@ -12,6 +12,7 @@
 import subprocess
 import textwrap
 import json
+import raven
```

Comment:
Sorry to be a pain, but this should go in the second block of imports, as these are organised as:

```
system imports

vendor imports

local imports
```